superset/assets/visualizations/nvd3_vis.js

```diff
@@ -476,6 +477,78 @@ function nvd3Vis(slice, payload) {
       .attr('height', height)
       .attr('width', width)
       .call(chart);
+
+      // add annotation_layer
+      if (isTimeSeries && payload.annotations.length) {
+        const tip = d3tip()
```

Comment:
Do these tooltips work when we are using "Rich Tooltips" (useInteractiveGuideline) How does that look?